pkg/cluster/internal/providers/podman/node.go

```diff
@@ -149,3 +150,7 @@ func (c *nodeCmd) SetStderr(w io.Writer) exec.Cmd {
 	c.stderr = w
 	return c
 }
+
+func (n *node) SerialLogs(writer io.Writer) error {
+	return common.ExecToWriter(exec.Command("podman", "logs", n.name), writer)
```

Comment:
```suggestion
	return exec.Command("podman", "logs", n.name).SetStdout(w).SetStderr(w).Run()
```

(also `writer` can just be `w`, there's no extra info in the variable name since it's just the type, e.g. https://golang.org/pkg/io/#WriteString)

Reply:
done